compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposeSwingLayer.desktop.kt

```diff
@@ -35,14 +35,53 @@ import org.jetbrains.skiko.swing.SkiaSwingLayer
  * Provides [component] that can be used as a Swing component.
  * Content set in [setContent] will be drawn on this [component].
  *
- * [ComposeSwingLayer] provides smooth integration with Swing, so z-ordering, double-buffering etc. from Swing will be taken into account.
+ * [SwingComposeBridge] provides smooth integration with Swing, so z-ordering, double-buffering etc. from Swing will be taken into account.
  *
- * However, if smooth interop with Swing is not needed, consider using [androidx.compose.ui.awt.ComposeWindowLayer]
+ * However, if smooth interop with Swing is not needed, consider using [androidx.compose.ui.awt.WindowComposeBridge]
  */
-internal class ComposeSwingLayer(
+internal class SwingComposeBridge(
     private val skiaLayerAnalytics: SkiaLayerAnalytics
-) : ComposeLayer() {
-    private val _component = ComposeSwingSkiaLayer()
+) : ComposeBridge() {
+    /**
+     * See also backendLayer for standalone Compose in [androidx.compose.ui.awt.WindowComposeBridge]
+     */
+    @OptIn(ExperimentalSkikoApi::class)
+    private val _component =
```

Comment:
Why we create an additional field?

Kotlin can behave well, if we just override `val component: SkiaSwingLayer`:

```
override val component = object : SkiaSwingLayer
```